code/modules/hydroponics/plant_mutations.dm

```diff
@@ -199,30 +203,39 @@
 // Eggplant Mutations
 
 /datum/plantmutation/eggplant/literal
-	name = "free range eggplant"
+	name = "Free-Range Eggplant"
 	name_prefix = "free range "
-	iconmod = "eggs"
+	iconmod = "EggplantEggs"
 	crop = /obj/item/reagent_containers/food/snacks/ingredient/egg
 
 // Wheat Mutations
 
 /datum/plantmutation/wheat/durum
-	name = "durum wheat"
+	name = "Durum Wheat"
 	name_prefix = "durum "
 	crop = /obj/item/plant/wheat/durum
 
 /datum/plantmutation/wheat/steelwheat
 	name = "steel wheat"
 	name_prefix = "steel "
-	iconmod = "steelwheat"
+	iconmod = "WheatSteel"
 	assoc_reagents = list("iron")
 	crop = /obj/item/plant/wheat/metal
 
+// Bean Mutations
+
+/datum/plantmutation/beans/jelly
+	name = "Jelly Bean"
+	name_prefix = "jelly"
+	iconmod = "BeanJelly"
+	assoc_reagents = list("VHFCS")
+	crop = /obj/item/reagent_containers/food/snacks/candy/everyflavor
```

Comment:
I feel like this got sneaked in with the rest of the changes, and is probably deserving of its own PR rather than being in here? It definitely has some balance implications.

Reply:
I can slap an ASS_JAM ifdef on it? or alternatively i can just comment it out

Reply:
Personally, I would remove from this PR (i.e. tack on a commit to remove those lines) and throw up another one with it in. This PR is too much good content to taint with Ass Jam.

Reply:
pahaha ok ill remove it for this one - i wont remove the icons because theyre a pain to set up if thats ok

Reply:
Resolved in latest commit.